### PR TITLE
Add trigger for receipting perf test

### PR DESCRIPTION
**Pull request by hbrace**
# Motivation and Context

Adds [receipting performance test](https://github.com/ONSdigital/census-rm-performance-tests/pull/28) trigger
# What has changed

Job to trigger receipting performance test
# How to test?

Check it all looks good
# Links

https://trello.com/c/7GLcz4tB/625-add-performance-test-for-receipting-8
https://github.com/ONSdigital/census-rm-performance-tests/pull/28
# Screenshots (if appropriate):

### CODE REVIEW

**Comment by hbrace** [comment]
[Add Performance Test for Receipting (8)](https://trello.com/c/7GLcz4tB/625-add-performance-test-for-receipting-8)

File: pipelines/performance-tests-pipeline.yml [hunk 1]
```diff
@@ -308,6 +308,35 @@ jobs:
 
               echo "\"@three-hundred-and-fifty-thousand\"" | gsutil cp - gs://census-rm-performance-sample-files/target-scenario-tag.txt || true
 
+- name: "Select Test - Receipting"
+  plan:
+  - *slack_performance_tests_started
```
Comment (NickGrantX) [inline pipelines/performance-tests-pipeline.yml]: This is a bit of a bug that I introduced. This should be a totally different slack alert, but I messed up. If you like, you can fix it!

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

File: pipelines/performance-tests-pipeline.yml (hunk 1 above)
Comment (hbrace) [reply pipelines/performance-tests-pipeline.yml]: Got rid of these slack alerts

**Review by gavinleeedwards — APPROVED** [review]

**Pull request merged**